scripts/get-stats.sh

```diff
@@ -21,12 +21,17 @@ set -x
 
 DURATION_TIME=${DURATION_TIME:-10}
 
-NODE_NAMES=$(/usr/local/bin/oc get node -o name -l node-role.kubernetes.io/worker)
+if ! [ -f "/home/zuul/bin/oc" ]; then
```

Comment:
Instead of doing that, I'd recommend something like:
```
PATH=$PATH:/home/zuul/bin
```
Then you would be able to just call `oc` command without the full path.